src/XMLSecurityKey.php

```diff
@@ -59,15 +59,37 @@ class XMLSecurityKey
     const RSA_SHA512 = 'http://www.w3.org/2001/04/xmldsig-more#rsa-sha512';
     const HMAC_SHA1 = 'http://www.w3.org/2000/09/xmldsig#hmac-sha1';
 
+    /** @var array */
     private $cryptParams = array();
+
+    /** @var int|string */
     public $type = 0;
+
+    /** @var mixed|null */
     public $key = null;
+
+    /** @var string  */
     public $passphrase = "";
+
+    /** @var string  */
     public $iv = null;
+
+    /** @var string */
     public $name = null;
+
+    /**
+     * @var mixed|null
+     * @deprecated
```

Comment:
I don't see any use in an empty method - without overriding another implementation - or do you mean that there should be a comment why it is deprecated?